lib/drivers/dexcomDriver.js

```diff
@@ -285,12 +286,19 @@ module.exports = function (config) {
     var all = [];
     var ctr = 0;
 
-    var format = 'iisbs';
-    var recordformat = ['systemSeconds', 'displaySeconds', 'glucose', 'trendArrow', 'crc'];
+    var format;
+    var recordformat;
+    debug('EGV header revision:',header.revision);
```

Comment:
Yes, this will be useful for debugging in case there are unexpected revision numbers that should pass (In the unlikely case that a G4 receiver has a revision number larger than 4.)